manager/config/control-plane-security/authentication.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 IBM Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: "security.istio.io/v1beta1"
+# This policy enables mutual TLS traffic between every two workloads that have sidecar.
+kind: "PeerAuthentication"
+metadata:
+  name: "premissive-mtls-in-control-plane"
```

Comment:
I see that other kustomize resources specify `namespace: system` (which is later altered automatically with the added prefix `m4d-`). This is relevant to all of the resources you added.

Reply:
Sure will add it. I used namespace: m4d-system in kustomization.yaml and missed it. Will change that.